src/_ert_job_runner/job.py

```diff
@@ -76,23 +76,17 @@ def run(self):
                 stat = os.stat(target_file)
                 target_file_mtime = stat.st_mtime_ns
 
-        exec_env = self.job_data.get("exec_env")
-        if exec_env:
-            exec_name, _ = os.path.splitext(
-                os.path.basename(self.job_data.get("executable"))
-            )
-            with open(f"{exec_name}_exec_env.json", "w", encoding="utf-8") as f_handle:
-                f_handle.write(json.dumps(exec_env, indent=4))
-
         max_running_minutes = self.job_data.get("max_running_minutes")
         run_start_time = dt.now()
-
+        environment = self.job_data.get("environment")
+        if environment is not None:
+            environment = {**os.environ, **environment}
```

Comment:
So if the environment is not set, we won't pass os.environ?

Reply:
yes